### ISSUE TITLE
Allow choice of Pointer

### ISSUE THREAD

**Issue by Tarzan5** [CITGuru/PyInquirer#24]
For the list/checkbox types, the pointer is the unicode character U+276F. Windows command prompt seems to have an issue with displaying that character (see screenshot). Note: Notepad displays the symbol correctly using the same font, so it's not an issue with the font. Would be nice if we could choose the pointer symbol too, just like we can choose the 'qmark' symbol. Thanks.

![capture2](https://user-images.githubusercontent.com/44897144/48261804-43bb6b80-e446-11e8-9e79-d9a34c4d1ce6.png)

**Comment by Dauflo** [comment]
+1

**Comment by CITGuru** [comment]
Sorry for late response, I actually looked into this, I noticed that is the default pointer from `prompt_toolkit`

**Comment by rigogsilva** [comment]
Can you change the default?

**Comment by CITGuru** [comment]
@rigogsilva I am not sure, I did some research on it and I notice there was nothing like that. So I am trying to look at the `python-prompt-toolkit - 1.0`, if we could change this.

**Comment by CITGuru** [comment]
I dont think we could change it for now, If any one has an idea how to, you can suggest a bug fix

**Comment by BlackJack1412** [comment]
Hello, we can change the pointer by modifying
\u276f to other character
PyInquirer 1.0.3:
- list:
tokens.append((T.Pointer if selected else T, ' \u276f ' if selected else '   '))
- checkbox:
if pointed_at:
          tokens.append((T.Pointer, ' \u276f ', select_item))

**Comment by mateoglzc** [comment]
Hello Guys! I'm having the same problem as you. The list pointer is not showing up in my terminal :( I changed the symbol using the @BlackJack1412 solution, but has anyone implemented a solution and doesn't require a source code change?
